Ontology/Capability/Sensor/StateSensor/BinarySensor/BinarySensor.json

```diff
@@ -5,5 +5,17 @@
     "en": "Binary sensor"
   },
   "extends": "dtmi:digitaltwins:rec_3_3:device:StateSensor;1",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "lastValue",
+      "displayName": {
+        "en": "Last Value"
```

Comment:
For consistency with the rest of the ontology (and the underlying OWL models) this should be all lowercase, i.e., "last value"

Reply:
Got it, will modify